torch/csrc/jit/pickler.cpp

```diff
@@ -145,6 +154,28 @@ void Pickler::pushDevice(const IValue& ivalue) {
   }
 }
 
+#ifdef USE_DISTRIBUTED
+void Pickler::pushRRef(const IValue& ivalue) {
```

Comment:
I'm not very familiar with the semantics of RRef are but the changes to the pickler here look fine. Can you elaborate on what it means to serialize an RRef? Is it possible to load a model much later and have a dead reference or something?

Reply:
thanks for reviewing @driazati. We are implementing how to serialize RRef during RPC call only right now, we have not implemented how to save a model with RRef. 

When we make RPC call that takes RRef as args, we need to pickle RRef by passing a few IDs and type of the RRef over the wire, and then recreate the RRef from the received IDs and type during unpickling.

I will add more description in the summary